### ISSUE TITLE
feature request: verified ownership of assets.

### ISSUE THREAD

**Issue by jeroz1** [RavenProject/Ravencoin#384]
Including a hash-generator that someone can include in their medium (such as a website) and let RVN check whether that hash is inside that medium, so that it would generate verified ownership. 

I think this option should be available after asset creation. 

Example:
If I am the owner of SuperLemonade9000, I want people to know that I am the official owner of both the Asset and the company, I could link my website www.SL9000.com to the asset name SUPER_LEMONADE_9000 and the scammer with SUPRR_LEMOANADE_9000 has an unverified asset.

Initial idea was coined by [@]Hardman[#]0001 

Edit:
So in other words, proof of the fact that the creator of asset X belongs to the owner of website Y. Something like an SSH key pair. And a KISS feature in the wallet to do this and show token holders that the asset is verified. (and when you click or hover over it, with which website).

**Comment by TronBlack** [comment]
I like this idea.

One KISS option is to include a signature in the 
rvn.<domain>.com  TXT record for the domain name.

Steps:
Take your asset name, and put it in the RVN client, and sign it with key for the same address that issued the token.   File->'Sign message' where the message is the name of the token.

Any client could quickly verify the token name by looking at the last issuance address (in the case of re-issue), the token name, and the signature in the TXT record.   With those three pieces of information, "Verify Signature" will return a true/false.  Only the issuer would be able to create that signature and place it in the TXT record for the domain.

Might need the root domain name in the metadata for this to happen automatically.

**Comment by TronBlack** [comment]
Added domain to the metadata spec to be used for this purpose.



Sign asset name with issuer address.

Example:
Issued: BITACTIVATE to mqRiUsjJ7BGaztQ2exJTHbiX7DmmtAniiZ with metadata on testnet.  domain=bitactivate.com
Signed message BITACTIVATE with private key for mqRiUsjJ7BGaztQ2exJTHbiX7DmmtAniiZ
Put in TXT record for rvn.bitactivate.com
ICqPPxhNizqXcdocvsc3+Iq6FMMRL4j/y5niotdWI0aTK/XSqzc6eaAEcakXw6xb0OAWekq7lCPBSi6tm+e+5OE=

Return with:
nslookup -q=TXT rvn.bitactivate.com

**Comment by TronBlack** [comment]
Thank you for the ideas:
jeroz1 and [@]Hardman[#]0001

**Comment by jeroz1** [comment]
Thank you for picking it up! We cant beat scamming but we can make it a bit more difficult for them.

**Comment by TronBlack** [comment]
Solution presented.  Closing issue.
